src/OpenDiffix.Core.Tests/Analyzer.Tests.fs

```diff
@@ -337,99 +337,19 @@ type Tests(db: DBFixture) =
 
     result.Having |> should equal expected
 
+  // NOTE: We do QueryValidator testing in its respective test module. Here we just check, if it's invoked at all based
```

Comment:
I'm not super happy about this part, I was toying with the idea of splitting the `is-it-anonymizing` logic part out somewhere and test it properly. But then I realized, that we're doing some validations while analyzing, and there we don't discriminate on anonymizing/non-anonymizing, so possibly this requires a larger revamp.

For now keeping simple and leaving as is, with this comment.